src/cryptonote_core/sent_transition/sent_transition.cpp

```diff
@@ -464,14 +465,13 @@ void transition(
             continue;
 
         const auto& eth_addr = it->second;
-        unallocated[eth_addr] += oxen_to_sent(val);
+        unallocated[eth_addr] += oxen_to_sent(val.amount);
         log::debug(
                 logcat,
                 "oxen -> sent ({} -> {}) accrued unpaid oxen rewards: {}",
                 addr,
                 eth_addr,
-                val);
-        converted_rewards[oxen_addr] = val;
+                cryptonote::print_money(val.amount.to_coin()));
```

Comment:
Is there a reason here (and several other places using the same pattern) to not use just `val.amount` as the format argument, to format it via reward_money::to_string()?  `cryptonote::print_money(val.amount.to_coin())` as an argument means it *always* has to do string formatting, even if the log statement isn't going to be displayed, while passing a formattable type only formats it when needed.  The other difference is that print_money will drop off the subatomic digits, but I don't really see that as being needed in most of these log statements.